examples/yoti_example_flask/requirements.txt

```diff
@@ -24,8 +24,8 @@ python-dotenv==0.8.2
 requests==2.21.0
 six==1.11.0               # via cryptography, protobuf, pyopenssl
 urllib3==1.24.2
-werkzeug==0.14.1          # via flask
-yoti==2.8.0
+werkzeug==0.15.5          # via flask
+yoti==2.8.1
```

Comment:
Shouldn't this be 2.8.2 now?

Reply:
This can't be updated until 2.8.2 has been released on PyPi so once we have released it, we will update the example projects to use 2.8.2